Types.ts

```diff
@@ -106,5 +106,5 @@ export type Config = {
         ignorePrefix: string[];
     };
     vcPing?: { [key: string]: string };
-    modMail: Snowflake;
+    modMail?: Snowflake;
```

Comment:
Is this part of the upgrade?

Reply:
No but it's implemented in the file but not the types.
*it's a single character change that better reflects the implementation*